src/Core/Form/IdentifiableObject/DataProvider/BulkCombinationFormDataProvider.php

```diff
@@ -62,6 +62,9 @@ public function getDefaultData()
                 'ean_13' => '',
                 'isbn' => '',
             ],
+            'images' => [
+                'images' => [],
```

Comment:
Strange structure isn't it ? Is it due to some technical debts or something from domain i don't know ? :thinking:

Reply:
form structure forces us having such data. We could normalize it with additional callbacks in form, but that would probably be overhead